Backend/internal/auth/login.go

```diff
@@ -93,11 +98,14 @@ func LoginHandler(w http.ResponseWriter, r *http.Request) {
 		}
 
 		// else create a session if user's session isnt already created
-		// todo
-		//
-		// placeholder return success for now
-		// i know this is bad
-		throwErr(errors.New("success"), w)
+		_session.CreateSession(w, r, user.Email)
+
+		// will change to FRONTEND_URI soon
+		http.Redirect(w, r, "http://localhost:3000/dashboard", http.StatusMovedPermanently)
```

Comment:
switch statements require a break otherwise they will attempt to verify the next case